docker/Dockerfile

```diff
@@ -0,0 +1,34 @@
+FROM centos:7
+MAINTAINER Apache BookKeeper <dev@bookkeeper.apache.org>
+
+ARG BK_VERSION=4.4.0
+ARG DISTRO_NAME=bookkeeper-server-${BK_VERSION}-bin
+ARG ZK_VERSION=3.5.2-alpha
```

Comment:
It used to verify if zk is ready before do zk changes when deploy a new bookie. 
If zk is not ready, it will wait there until zk is OK.
```
until /opt/zk/bin/zkCli.sh -server $ZK_URL ls /; do sleep 2; done
```